Directory.Build.props

```diff
@@ -47,13 +47,20 @@
     <DefineConstants>$(DefineConstants);VS2017</DefineConstants>
   </PropertyGroup>
 
-  <PropertyGroup>
+  <PropertyGroup Condition=" '$(VsVimTargetVersion)' == '16.0' ">

```

Comment:
Please move the VS2019 changes into a separate PR. Want to keep this focused on the perf problem. Makes it easier to back port if it becomse necessary.

Reply:
Agree